### PR TITLE
Subscription update - Stripe

### LINKED ISSUE #343: Subscriptions
With the incoming changes in payment method through stripe, we will need to update the [Subscriptions and Invoicing](https://merginmaps.com/docs/manage/subscriptions/)  page, including: changing/cancelling a subscription, invoices, payment history. 

Docs to update:
- [ ] [Subscriptions and Invoicing](https://merginmaps.com/docs/manage/subscriptions/)
- [ ] [Home Page and Dashboard - Subscription](https://merginmaps.com/docs/manage/dashboard/#subscription)

### PR DESCRIPTION
**Pull request by alex-cit**
fix #343 update of [Subscriptions and Invoicing](https://merginmaps.com/docs/manage/subscriptions/) page
- updated info about how to manage subscriptions through stripe
- [How to request Academia or Non-profit plan](https://merginmaps.com/docs/manage/subscriptions/#how-to-request-academia-or-non-profit-plan) refers to the [pricing page](https://merginmaps.com/pricing) (not the dashboard)
- [Subscriptions and Invoicing](https://merginmaps.com/docs/manage/subscriptions/)  included [How to transfer ownership and billing for workspaces](https://merginmaps.com/docs/manage/subscriptions/#how-to-transfer-ownership-and-billing-for-workspaces) - transferring workspace ownership was moved to (Member Roles and Permissions)[https://merginmaps.com/docs/manage/permissions/] , changing billing details is included in the Subscriptions page
- new **Mergin Maps dashboard** component (refers to app.merginmaps.com)
- new **dashboard** component (refers to app.merginmaps.com)

### CODE REVIEW

**Comment by tomasMizera** [comment]
Ahoj @alex-cit, thanks for the PR!

> Do we want users to go to the dashboard and find the link to the payment portal or should they go there directly? (I used payments.merginmaps.com as a placeholder link)

They should go to the dashboard "subscription" page (app.merginmaps.com/subscription). `payments.mm...` links are generated on the fly and it might be a different link each time

> [Subscriptions and Invoicing](https://merginmaps.com/docs/manage/subscriptions/) includes [How to transfer ownership and billing for workspaces](https://merginmaps.com/docs/manage/subscriptions/#how-to-transfer-ownership-and-billing-for-workspaces) - do we want to move it to (Member Roles and Permissions)[https://merginmaps.com/docs/manage/permissions/] or (Workspaces)[https://merginmaps.com/docs/manage/workspaces/]? I think transferring workspace ownership and transferring billing will now be 2 different tasks

Good point! Let's separate the two topics and move "transfer ownership" to "member roles and permissions". The new section for billing can be called something like "Update billing details" with a note that the email there will receive all billing-related information like receipts, failed payments notifications and more.

**Comment by alex-cit** [comment]
@tomasMizera thanks for your comments, ready for review :)

File: src/manage/permissions.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Do you want to invite members or guests to your workspace? Visit [How to share p
 - **Reader**: users with **read** permission are able to see the list of projects, projects data and [history](./project-details.md)
 - **Writer**: in addition to the **Read** access, users can also do changes in the projects: [adding/removing/updating features](../field/input_features/), change [layer settings and project properties](../gis/features/). Users who are expected to actively contribute to the project need to have the **write** permission. 
 - **Admin**: in addition to the **Write** access, users with this level of access can [delete the project](./project-advanced/#delete-a-project) or [transfer](./project-advanced/#transfer-a-project) it to another workspace. Admins can also create new projects and manage workspace members.
-- **Owner**: in addition to the **Admin** permission, owners have access to [invoicing and subscription](../manage/subscriptions/) settings.
+- **Owner**: in addition to the **Admin** permission, owners have access to [invoicing and subscription](./subscriptions/) settings.
```
Comment (tomasMizera) [inline src/manage/permissions.md:22]:
```suggestion
- **Owner**: in addition to the **Admin** role, owners have access to [invoicing and subscription](./subscriptions/) settings.
```

File: src/manage/permissions.md [hunk 2]
```diff
@@ -38,6 +38,33 @@ Do you want to invite members or guests to your workspace? Visit [How to share p
 Members and their roles can be managed through <AppDomainNameLink /> in the [Members](./dashboard/#members) tab:
 ![Mergin Maps web members tab](./web-members.jpg "Mergin Maps web members tab")
 
+### How to transfer ownership of a workspace
+Ownership of a workspace can be transferred to another [workspace member](#workspace-member-roles). 
+
+Users with the **owner** member role have access to the **Subscriptions** in <AppDomainNameLink />. Every workspace has to have at least one owner, however, there can be multiple owners of one workspace.
```
Comment (tomasMizera) [inline src/manage/permissions.md:44]: How about
```suggestion
Users with the **owner** member role have access to the **Subscriptions** page in <DashboardShortName>. Every workspace has to have at least one owner, however, there can be multiple owners of one workspace.
```

File: src/.vuepress/components/DashboardLink.vue [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <a :href="`https://app.merginmaps.com/${id}`" target="_blank" class="nospellcheck">
+    <span v-if="desc" v-html="desc"></span>
+    <span v-else class="nospellcheck">Mergin Maps Dashboard</span>
+  </a>
+</template>
+
+<script>
+export default {
+  name: "Mergin Maps Dashboard",
```
Comment (tomasMizera) [inline src/.vuepress/components/DashboardLink.vue:10]: Shall we have a short name for this one? "Dashboard"

**Review by tomasMizera — APPROVED** [review]

**Comment by tomasMizera** [comment]
Great job, thanks! 🚀

**Pull request merged**